### ISSUE TITLE
Can it be used in the Chinese version?

### ISSUE THREAD

**Issue by Alianore** [SilentNightSound/GI-Model-Importer#67]
I tried to run on the YuanShen.exe instead of Genshin Impact.exe, but it will appear the following window and automatically close it after a few seconds.
![image](https://user-images.githubusercontent.com/92358695/217333682-b114205c-0c58-40b7-9e11-ad15a5bf178a.png)
It does not seem to play its due role.

**Comment by Alianore** [comment]
In all the tutorials, I only need to download 3dmigoto-GIMI-for-development.zip. Does it need other support?

**Comment by Alianore** [comment]
Everything is normal when the target is GenshinImpact.exe, but as long as I change the GeenshinImpact.exe in d3dx.ini to YuanShen.exe, the Loader will automatically close after a few seconds.

**Comment by ReBearRain** [comment]
I'm having the same problem and I've found some people are having the same problem, maybe it doesn't apply to the Chinese version?

**Comment by ReBearRain** [comment]
Perhaps there is another explanation that this is instead normal?

**Comment by Alianore** [comment]
Thank you for your suggestion. I asked the relevant questions in "General-ZH" earlier, but no one answered, and when I read the chat history, I found that someone asked the same question. The emergence of this problem may be because the process of the same name is running silently. When setting "Delay = -1", it can occasionally see this phenomenon in the display of "Loader". I can't confirm that it is from mihoyo or myself. In any case, I finally solved the problem by setting up "Delay = -1" because of reference to your suggestions. Thank you again for your help!
